addon_service/authorized_storage_account/serializers.py

```diff
@@ -95,8 +95,14 @@ def create(self, validated_data):
             authorized_account.initiate_oauth2_flow(
                 validated_data.get("authorized_scopes")
             )
+        elif external_service.credentials_format is CredentialsFormats.OAUTH1A:
+            authorized_account.initiate_oauth1_flow()
+            self.context["request"].session[
+                "oauth1a_account_id"
+            ] = authorized_account.pk
```

Comment:
Do we have to store the `authorized_account.pk` in the session? Is there anything in the payload of the request to the callback url that can be used to identify which authorized account this is?